core/src/test/scala/unit/kafka/log/UnifiedLogTest.scala

```diff
@@ -71,6 +72,13 @@ class UnifiedLogTest {
   @AfterEach
   def tearDown(): Unit = {
     brokerTopicStats.close()
+    try {
+      log.close()
```

Comment:
perhaps worth checking that (log != null) otherwise we may have null pointer exception here if the test exits before creating a log object

Reply:
should be fixed now.